### ISSUE TITLE
[Feature Request] Provide Batch convert options for adding and removing color from all lines(e.g. in srt files)

### ISSUE THREAD

**Issue by Hlsgs** [SubtitleEdit/subtitleedit#6448]
For an srt that already has all lines colored, I gather that one can clean them up in Batch using the `Convert colors to dialog` option with only `Remove color tags` selected. However, it would be more intuitive if the `Remove formatting tags` option had sub-options like the context menu `Remove formatting`.

Conversly, there is no current way that I am aware of for Batch adding color to all lines in `srt` files. One can only open them one-by-one, select all lines and use the `Color` context menu option. This need arises from the popularisation of HDR content, which requires a darker tone for subs compared to SDR.

If both these options were available in CLI, that would be even better. Speaking of which, quick question: do the CLI `/convert` options, for example `/RemoveTextForHI`, follow their respective options that have been previously set in the `Batch convert` GUI, of do they just use their default parameters?

**Comment by niksedk** [comment]
You can do this with "Multiple replace" and regular expressions

**Comment by Hlsgs** [comment]
> You can do this with "Multiple replace" and regular expressions

True, but, using myself as an example here, while I could probably get the regex for tag removal right, I can't say the same about the one adding the font tags. Just gave it a try with `(.*)` replaced with `<font color="#DDDDDD">$1</font>` and that results in one font tag per line in multiline plus an additional empty `<font color="#DDDDDD"></font>` at the end of each line. And I'm sure there are tons of edge cases, like what to do when there are already some color tags present etc. Whereas selecting all lines and using the `Color` context menu option always gets things right. And I would venture to guess that most users would't even know where to begin with regular expressions. And then there's CLI: how would one pass the rules to `/multiplereplace`?

Anyway, if you feel that this functionality is not worth adding, I'm going to have to ask: what would be the regex expressions covering all cases that you're aware of for both addition and removal of font tags for all lines?
